### DISCUSSION TITLE
Install without docker?

### DISCUSSION THREAD

**Discussion by pete2583** [RandomNinjaAtk/docker-lidarr-extended#234] [Q&A]
So, I don't use docker on my main server but would prefer this on the server as it can handle a bit more. 

Is there a way to implement this version into a "standard" Lidarr install? 

Thanks for any help or advice you can give me.

**Comment by RandomNinjaAtk** [comment]
If it's a Linux system, this is just a series of bash scripts. So you'd have to install the pre-reqs and could download the scripts manually and use them. There is no guide on how to do that, but completely possible. 

If your on windows, you can't do that though...

**Reply by pete2583** [reply]
Yeah, I'm on Linux. 

So, download the scripts then set a cronjob to run them every 15 minutes or something? Would it really be that simple?!

**Reply by RandomNinjaAtk** [reply]
You'll need to look through the docker file to install all the required packages, but yes.

You'll also need to likely modify the scripts to set the variables for them manually since you won't be using docker ENV for the variables...

**Reply by pete2583** [reply]
Yeah, I figured I would have you edit the scripts. 

Ok, might give that a shot later. Cheers
